### ISSUE TITLE
I cannot access the control pannel when connected to the vpn

### ISSUE THREAD

**Issue by pilpil261** [wg-easy/wg-easy#539]
**Describe the bug**
I installed ubuntu server LTS 64bit on my Raspberry Pi 4. I want to run Wireguard in order to access my other applications like Nextcloud on my Pi. I messed around with trying to install Wireguard the normal way but ended up using this because this was the only way I even able to connect to my vpn. However. When i connect to the VPN, no matter if I am on my home network or not, it won't let me access the web gui of wg-easy and won't let me access my portainer web gui.
**To Reproduce**
1. I simply installed wg-easy using the guide on the github page. 
2. I copy-pasted everything. 
3. I changed my host ip to 192.168.1.104, which is the static ip I gave my pi in my home network.  
4. I forwarded the port 51820, using 51820 as external and internal port, udp protocol and 192.168.1.104 as internal ip.
5. I accessed the web-gui and created 2 new profiles, one for my phone and one for my Desktop.
6. I connected on my Desktop, got disconnected from my ssh connection to my raspberry pi and couldn't access wg-easy web gui and portainer web gui anymore.
7. I connected on my phone, using my mobile data and not my home wifi, which resulted in the same problem.
8.  I used "docker stop wg-easy
docker rm wg-easy
docker pull weejewel/wg-easy"
9. I changed the ip-address of my pi to 192.168.2.104 in my router and in the docker run-d \... command.
10. I tried forwarding the port with the new IP-Address which gave me an error saying something along the lines of: "The IP-Address does not exist in the same subnet as the LAN-IP-Address."
11. go to github, look at issues, don't find anything, write here.

**Expected behavior**
I expected to be able to access my portainer and wg-easy web gui, and later on my nextcloud over the same vpn connection.

**Desktop (please complete the following information):**
 - OS: VPN-Host: Ubuntu Server LTS 20.04 (64 bit), Client 1: Windows 10
 - Browser Brave
 - Version latest

**Smartphone (please complete the following information):**
 - Device: Google Pixel 7
 - OS: Graphene-OS
 - Browser Brave
 - Version Latest

**Comment by VATICAN-PSYCHO** [comment]
This isn't bug or any problem related to wg-easy.
This may be happening because of (one or more) your device isn't connected to WG. In order for VPN (any, not only) to _work_ they must (the simplest/ the most popular case) have access to Internet, simple. You can create new docker network or join wg-easy container to existing (i don't remember which one) with internet access.

Another thing is that you to add appropriate routes on your Pi. The easiest way would drawing simple diagram/schema of your environment with all networks, attached devices (and IPs on each used interface) and pinging them one by one from several nodes in your network.

**Comment by Archontes** [comment]
I can corroborate this. When connected to wg-easy over 5g, I cannot navigate to the web UI, though I can when connected via Wi-Fi. I know the internet and vpn are working because my Jellyfin instance is functional.

**Comment by SeryiBaran** [comment]
same trouble. After connection to server, ssh access drops.

**Comment by SeryiBaran** [comment]
> same trouble. After connection to server, ssh access drops.

i'm moved to pivpn.

**Comment by voltane** [comment]
Moving to another solution does not seems to be an issue resolver :D Anyways, i'm also having same issue (TrueNAS -> TrueCharts -> wg-easy). UI woks fine when inside local network, but not working at all when connected through VPN. Other services are available.

**Comment by acul009** [comment]
I'm having a similar problem, but to a greater extend. I'm running guacamole and wg-easy on the same host. Guacamole needs to be accessible via the normal LAN too. When I use the LAN-IP when connected via Wireguard, I can't reach ANY published ports.

But I just noticed, that it works if I disable ufw. Unfortunately I'm still not that good with the Linux firewall, but maybe that information helps someone.

**Comment by JohannesDev** [comment]
Are there any updates on this? Has anyone actually managed to solve this? I can access the Dashboard of wg-easy perfectly as long as I am in my local network. When I use my phone (which is outside my local network but connected to Wireguard) I can access all local services fine except for the Wireguard Dashboard for some reason.

**Comment by Goosegit11** [comment]
Maybe this can be helpful:

https://lainchan.org/sec/res/21833.html

A ton of debugging info, I think. (Yep, OP is me)

**Comment by Goosegit11** [comment]
.

**Comment by dkgitdev** [comment]
Alright, seems like I've found a workaround.

I've been having the same issue. I've set up nginx to proxy_pass from my domain to webui.

It just showed the blank page on nginx proxy, but everything was fine on direct connection.

Using nginx page, I switched to console tab in dev tools and noticed issues with [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) which I didn't have on direct connection.

After some googling and trial-and-error, I've managed to find minimum configuration to add to nginx `location /` block:
```
add_header Content-Security-Policy "script-src 'self' 'unsafe-eval'" always;
```

And it worked for me :D

It might worth mentioning this in readme of this repo and / or arrange server to adds this header on a response, but I'm unable to help you with that, sorry :(
